sources/CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 # Create the library.
 add_library(${PROJECT_NAME} STATIC)
-add_library(oclero::${PROJECT_NAME} ALIAS ${PROJECT_NAME})
```

Comment:
Using an ALIAS is recommended for using a package as a CMake sub-project. Removing this line may break existing users.
If you want the package to migrate to a different *exported* target name, add an ALIAS for that name here, for future alignment of subproject usage and external usage.

Reply:
This is up to @oclero .

I removed it for the same reasons we did in `qlementine`. We were okay with this breaking change

Reply:
Yes we can remove the alias. The lib is only used by a few projects at the moment.